src/routes/index.svelte.md

```diff
@@ -101,6 +106,10 @@ blocked again. This may have been the cause for the day's earlier outage.
 
 While these incidents were in no way intentional, it highlights the poor QA testing that Manjaro performs. This has happened on two separate occasions in less than two years.
 
+## Miscellaneous
+
+[Their system update script runs `rm` on the lockfile mid-transaction.](https://gitlab.manjaro.org/packages/core/manjaro-system/blob/3b806753e245b7ec7e18bb674e916e28d751a429/manjaro-update-system.sh#L45(https://archive.fo/dofw8)).
```

Comment:
a little more context on this can be given -- maybe something like *why* this is bad